pkg/authorization/authorizer/messages.go

```diff
@@ -50,6 +50,9 @@ func NewForbiddenMessageResolver(projectRequestForbiddenTemplate string) *Forbid
 	}
```

Comment:
You should do a `git revert` on the merge commit of Clayton's revert to generate this